packages/airnode-deployer/config/secrets.example.env

```diff
@@ -7,5 +7,5 @@ HEARTBEAT_API_KEY=00000000-0000-0000-0000-000000000000
 HEARTBEAT_ID=
 HEARTBEAT_URL=
 
-HTTP_GATEWAY_API_KEY=00000000-0000-0000-0000-000000000000
-HTTP_SIGNED_DATA_GATEWAY_API_KEY=00000000-0000-0000-0000-000000000000
+HTTP_GATEWAY_API_KEY=12345678-0000-0000-0000-000000000000
```

Comment:
They are probably not because we do not have a strict validation - e.g. we do check mnemonic is a string, but not whether it's a valid mnemonic. If we want to verify that the configs are correct then it's more future proof to have the values here.